release notes/v0.5.0.md

```diff
@@ -0,0 +1,125 @@
+xk6-browser v0.5.0 is here! :tada:
+
+In this minor release we have attempted to make the application more stable by tackling the most problematic race conditions we have come across. The most notable changes are the following two **breaking changes**:
+
+- `click` and `waitForNavigation` are now asynchronous, although we do not support the `async` keyword yet. We plan to make more of our apis asynchronous in the next release.
+
+- We have a new `Chromium` exported `BrowserType` which is to be used to launch a new instance of the `chromium` browser, and `launch` is no longer supported.
+
+
+## Bugs fixed
+
+- `Page.goto()` race condition, which was mainly found to occur in CI, looks to have been fixed. ([#480](https://github.com/grafana/xk6-browser/pull/480))
+
+- Fix nil panic race in `FrameManager.waitForFrameNavigation`. ([#500](https://github.com/grafana/xk6-browser/pull/500), [#501](https://github.com/grafana/xk6-browser/pull/501), [#508](https://github.com/grafana/xk6-browser/pull/508))
+
+
+## Improvements
+
+- We've made more changes to out log messages to make them more concise and user friendlier. ([#438](https://github.com/grafana/xk6-browser/pull/438))
+
+- `Chromium` is now an importable type which is the defacto method of launching tests using `Chromium` (which is still the only supported browser type). This is a **breaking change** which supersedes the `launch` function. The same options are supported in `Chromium`'s `launch` method as in the previous `launch` function. ([#462](https://github.com/grafana/xk6-browser/pull/462), [#515](https://github.com/grafana/xk6-browser/pull/515))
+
+  Here's an example of how to work with the `Chromium` type:
+
+  ```js
+  import { chromium } from 'k6/x/browser';
+
+  export default function () {
+    const browser = chromium.launch({
+      headless: false,
+    });
+    const context = browser.newContext();
+    const page = context.newPage();
+    const res = page.goto('https://test.k6.io/');
+    page.close();
+    browser.close();
+  }
+  ```
+
+  Here's how we used to launch the `Chromium` browser: 
+
+  ```js
+  import launcher from 'k6/x/browser';
+
+  export default function () {
+    const browser = launcher.launch('chromium', {
+      headless: false,
+    });
+    const context = browser.newContext();
+    const page = context.newPage();
+    const res = page.goto('https://test.k6.io/');
+    page.close();
+    browser.close();
```

Comment:
Instead of full examples, I think we can shorten these to just showcase the import and `launch()` calls, to not clutter the doc too much. Same for the `click()` and `waitForNavigation()` examples.

I'll push a change in a bit.

Reply:
Done in 74b363a.